### PR TITLE
Fixing PDF view/download file tests.(DHH-750)

### PR DESCRIPTION
**Pull request by hweng**

### CODE REVIEW

File: spec/features/dams_object_spec.rb [hunk 1]
```diff
@@ -427,6 +421,28 @@
     end
   end
 
+  describe "Curator PDF Viewer" do
```
Comment (mcritchlow) [inline spec/features/dams_object_spec.rb]: @hweng this is excellent, thanks for doing this so quickly. the only issue i notice is that I don't believe this feature is limited to curators, based on discussion in https://lib-jira.ucsd.edu:8443/browse/DHH-714 was there something you read in the ticket that called out curators specifically? if so, we should confirm with Gabriela

**Comment by hweng** [comment]
@mcritchlow, let me confirm with Gabriela, thanks!

**Comment by mcritchlow** [comment]
@hweng looks good! will merge once bamboo finishes running the tests

**Comment by mcritchlow** [comment]
bamboo is :smile: again. thanks @hweng !

**Pull request merged**